fp_output:
    formats: [
              "ODOMETRY", "LLH", "ODOMENU", "ODOMSH", "RAWIMU", "CORRIMU", "GNSSANT", "GNSSCORR", "TEXT", "TF", # FP_A
              "GPGGA", "GPGLL", "GNGSA", "GPGST", "GPHDT", "GPRMC", "GPVTG", "GPZDA", # NMEA
              #"GPGSV", "GAGSV", "GBGSV", "GLGSV" # GXGSV
             ]
    type: serial
    port: "/dev/ttyUSB0"
    serial_baudrate: 115200
    rate: 200
    reconnect_delay: 5.0 # wait time in [s] until retry connection

customer_input:
    speed_topic: "/fixposition/speed"
